css/main.css

```diff
@@ -239,7 +239,9 @@ p.paragraph {
     width: 100%;
 }
 .container-lrg {
-    max-width: 1080px;
+  align-items: baseline;
```

Comment:
Shouldn't be overriding anything in a utility class like this - make a new baseline class if you need this.